src/gluonts/support/pandas.py

```diff
@@ -11,13 +11,31 @@
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
 
+from typing import Dict
 
+import numpy as np
 import pandas as pd
 
 
 def frequency_add(ts: pd.Timestamp, amount: int) -> pd.Timestamp:
     return ts + ts.freq * amount
 
 
-def forecast_start(entry):
-    return frequency_add(entry["start"], len(entry["target"]))
+def forecast_start(entry: Dict) -> pd.Timestamp:
+    """
+    Determines the forecast start date from the start date and target time series provided in `entry`.
+
+    Supports multivariate target as well and the last dimension is assumed to be the target dimension.
+
+    Parameters
+    ----------
+    entry
+        Dictionary containing `start` and `target` time series.
+
+    Returns
+    -------
+        Timestamp
+            Forecast start date.
+
+    """
+    return frequency_add(entry["start"], np.shape(entry["target"])[-1])
```

Comment:
Minor: how about making the time axis configurable? With default `-1`

Reply:
I addressed all comments including this one.. this is hidden maybe because of resolving conflicts manually as the old `support` package is removed. It's [here](https://github.com/awslabs/gluon-ts/pull/1685/files#diff-07bbcc5e4c37496a9b8ef2a3bf07961fc70ff34599d14052434b657f4db15177R53).